hnsfuncs.go

```diff
@@ -53,6 +57,10 @@ type HNSEndpoint struct {
 	Policies           []json.RawMessage `json:",omitempty"`
 	MacAddress         string            `json:",omitempty"`
 	IPAddress          net.IP            `json:",omitempty"`
```

Comment:
I don't think we want to limit ourselves here. Can you follow up internally on what the roadmap is for full IPv4+IPv6 support?